src/index.js

```diff
@@ -0,0 +1,29 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+
+const app = express();
+
+const port = process.env.PORT || 3000;
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.get('/', (req, res) => res.status(200).json({ 
+    message: 'Welcome to orcas-phanthom backend site' 
+}));
+
+
+app.use(async (req, res) => {
+    const error = await new Error('Not found');
+    error.status = 404;
+    res.send({status: error.status, message: error.message});
+});
+
+
+app.listen(port, console.log(`server has started on port ${port}`));
+
+export default app;
```

Comment:
anable eslint and errors like 'no new line at the end of the file' will go way

Reply:
It's Okay now